Add excluded_types to link documentation (#719)
sulu-io/sulu-docs: reference/content-types/link.rst

      - collection
      - List of available types in the dropdown.
        Default: All registered link types
    * - excluded_types
      - collection

>       - List of types which should not show in the dropdown.

Reviewer: ```suggestion
      - List of types that should be excluded in the dropdown.
```
